src/data-workspace/section-form/section.js

```diff
@@ -38,7 +38,7 @@ export const SectionFormSection = ({ section, globalFilterText }) => {
 
     const maxColumnsInSection = Math.max(
         ...groupedDataElements.map(
-            (grp) => grp.categoryCombo.categoryOptionCombos.length
+            (grp) => grp.categoryCombo.categoryOptionCombos?.length || 1
```

Comment:
Same as in comment above

Reply:
I think it should stay here though. This is basically just defensive coding, and means if no group has the `categoryOptionCombos` due to some weird bug, we still get one column. This could however probably be moved to outside of the map, So its `Math.max(...) || 1`.

Reply:
I see, maybe then this expresses the intention a bit better:

```js
    // Never less than 1
    const maxColumnsInSection = Math.max(
        ...groupedDataElements.map(
            (grp) => grp.categoryCombo.categoryOptionCombos?.length
        ),
        1
    )
```

Reply:
There's a `ignoreDefaultValues` option in the magic number rule set, so I'd say you're right, this does make sense here